app/lib/api/auth.ts

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+
+const getRedirectUri = () => {
+    if (!process.browser) {
```

Comment:
It sometimes happened when compiling the pages for some reason, I assume it wanted to compile this method directly, meaning it didn't happen in the browser, so it failed because window was not defined yet

Reply:
Huh, interesting. Just wanted to make sure this isn't something we expect to happen on prod 😄

Reply:
I don't suspect it will, `process.browser` should always be true when in the browser. 

I found this thread, which covers a related issue: https://github.com/vercel/next.js/issues/5354#issuecomment-520305040
In that thread it was also noted that `!process.browser` is deprecated, I'll change it to `typeof window === "undefined"`